coding_standards.md

```diff
@@ -0,0 +1,104 @@
+# Coding Standards/Guidelines
+
+NUCore is an old project and has passed through many hands. There are long controller methods, complex view logic, and odd routing. It began as a Rails 2.X project and pieces of that legacy remain. Many of the things that you may consider as bad may have even been considered best practices at the time. Give the past developers some credit. Future you may have similar things to say about what you're doing right now.
+
+Times have changed, styles and best practices have evolved, and we want to keep the codebase healthy. That means cleaning things up and refactoring as needed. We can't fix everything at once, but we can make things better one little piece at a time.
+
+If you see a nasty or incomprehensible method, see if you can refactor it to make it more understandable. If you're trying to make a change and it's hard, [refactor to make the change easy](https://twitter.com/kentbeck/status/250733358307500032). 
+
+But do consider the value vs effort. If it's a bug fix where you have `>` instead of `>=`, it might not be worth spending four hours refactoring. But if you're working within a nasty method and you know there's constant churn on that class, it's likely worth it.
+
+The golden rule: **when working on a piece of code, _leave it nicer than when you found it_**.
+
+Here are some general guidelines:
+
+## Pull Requests
+
+**Do not commit to master!**
+
+All changes should go through a Pull Request and be reviewed prior to merging.
```

Comment:
Because I'm fully in favor of squashing commits when merging into master, I tend to lean towards people merging their own PRs once they've gotten the requisite :+1: